src/Orleans.Core/Async/AsyncExecutorWithRetries.cs

```diff
@@ -6,7 +6,7 @@
 namespace Orleans
 {
     /// <summary>
-    /// This class a convinent utiliity class to execute a certain asyncronous function with retires,
+    /// This class a convenient utility class to execute a certain asynchronous function with retires,
```

Comment:
nit: retries is also spelled incorrectly.

Reply:
Thank ya' @philrowan-mtb ! :+1: